_pytest/compat.py

```diff
@@ -146,13 +145,7 @@ def _ascii_escaped(val):
 
         """
         if isinstance(val, bytes):
-            if val:
```

Comment:
I think we should fallback to the old code in Python 3.3 and Python 3.4. It will make the current code uglier of course, but at least when we drop 3.4 we will get the chance of getting the better version.

If we don't merge this I fear we end up just forgetting about it.

Reply:
i will declare the a function sometime later to handle the detail